Base/Python/slicer/parameterNodeWrapper/serializers.py

```diff
@@ -1039,6 +1043,7 @@ class AnySerializer(Serializer):
       classes that were created in functions.
     - Can never cache.
```

Comment:
I wonder if it is possible to cache the _serializer_, so it doesn't need to be recreated on each read/write. If the name and module of the object's type don't change, the chosen serializer doesn't need to change either and so could persist. That might facilitate caching values in _certain_ circumstances, when the AnySerializer is used with the same type and serializer for that type supports caching.

That might be more difficult to implement than I'd expect, so I defer to you @Connor-Bowley on how feasible you think that would be.

Reply:
After some more thought I don't think this is a good idea. It is entirely incompatible with use cases on containers like `list[Any]` or un-annotated types like `dict`, etc.